draft-ietf-roll-capabilities.xml

```diff
@@ -237,9 +240,9 @@
                 </t>
                 <t>
                     Capabilities handshake could be optionally added with
-                    existing MOPs. Capabilities been optional in nature could
+                    existing MOPs. Capabilities, being optional in nature, could
                     be put to use with existing MOPs. Capabilities and
-                    MOP-extension is mutually independent i.e. a DIO can have a
+                    MOP-extension are mutually independent i.e. a DIO can have a
```

Comment:
Maybe I should simply remove MOP-Extension from here (remove this sentence). With the document split up it is no longer relevant.